test/cluster/test_read_repair.py

```diff
@@ -309,13 +310,13 @@ def check_rows(cql: Session, host: Host, expected_live_rows: row_set) -> None:
 
 
 @pytest.mark.asyncio
+@skip_mode('release', 'error injections are not supported in release mode')
 async def test_read_repair_with_trace_logging(request, manager):
     logger.info("Creating a new cluster")
-    cmdline = ["--hinted-handoff-enabled", "0", "--logger-log-level", "mutation_data=trace"]
+    cmdline = ["--hinted-handoff-enabled", "0", "--logger-log-level", "mutation_data=trace:debug_error_injection=trace"]
```

Comment:
Just curious -- why is trace-level required for error-injection logger?

Reply:
So I can see in the logs that my injection point was indeed enabled as the test requested.
It generates very little extra traffic, so I think it is fine to keep it in.